src/app/shared/issue-tables/issue-paginator.ts

```diff
@@ -5,7 +5,7 @@ export function paginateData(paginator: MatPaginator, data: Issue[]): Issue[] {
   paginator.length = data.length;
   let result = getDataForPage(paginator.pageIndex, paginator.pageSize, data);
   if (result.length === 0) {
-    paginator.pageIndex -= 1;
+    paginator.pageIndex = paginator.pageIndex - 1;
```

Comment:
Were there some issues with the original update to the page index?

Reply:
i got an error `TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.`